src/libs/OptionsListUtils.ts

```diff
@@ -1837,12 +1836,8 @@ function getShareDestinationOptions(
  * @param member - personalDetails or userToInvite
  * @param config - keys to overwrite the default values
  */
-function formatMemberForList(member: ReportUtils.OptionData, config: ReportUtils.OptionData | EmptyObject = {}): MemberForList | undefined {
-    if (!member) {
```

Comment:
do we have any case where this could error?

Reply:
Same as above, every usage of this function has member of ReportUtils.OptionData type, not undefined or null, so we don't need this check. Future usages are going to be checked by typecheck.